CppImport/CppImport.pro

```diff
@@ -53,7 +53,21 @@ HEADERS += src/precompiled.h \
     src/manager/TranslateManager.h \
     src/TranslateFrontendAction.h \
     src/CppImportPlugin.h \
-    src/comments/CommentParser.h
+	src/comments/CommentParser.h \
```

Comment:
It's a minor thing, but could you please change the indentation to match the rest of the file (spaces instead of tabs)